### PR TITLE
- Update maximum MTU for 10Gb NICs

### PR DESCRIPTION
**Pull request by darkfires**
10Gb switches use 9214 MTU on the interfaces vs 9000 for most 1Gb

### CODE REVIEW

File: src/www/interfaces.php [hunk 1]
```diff
@@ -908,8 +908,8 @@ interface_sync_wireless_clones($a_interfaces[$if], false);
             $input_errors[] = gettext("A valid MAC address must be specified.");
         }
         if (!empty($pconfig['mtu'])) {
-            if ($pconfig['mtu'] < 576 || $pconfig['mtu'] > 9000) {
-                $input_errors[] = gettext("The MTU must be greater than 576 bytes and less than 9000.");
+            if ($pconfig['mtu'] < 576 || $pconfig['mtu'] > 9214) {
+                $input_errors[] = gettext("The MTU must be greater than 576 bytes and less than 9214.");
```
Comment (fichtner) [inline src/www/interfaces.php]: for not running the risk of bouncing translation strings all the time on updates it should use sprintf %s for both the min and the max value

File: src/www/interfaces_lagg_edit.php [hunk 2]
```diff
@@ -115,8 +115,8 @@ function available_interfaces($selected_id=null)
     if (!in_array($pconfig['proto'], $laggprotos)) {
         $input_errors[] = gettext("Protocol supplied is invalid");
     }
-    if (!empty($pconfig['mtu']) && ($pconfig['mtu'] < 576 || $pconfig['mtu'] > 9000)) {
-        $input_errors[] = gettext("The MTU must be greater than 576 bytes and less than 9000.");
+    if (!empty($pconfig['mtu']) && ($pconfig['mtu'] < 576 || $pconfig['mtu'] > 9214)) {
+        $input_errors[] = gettext("The MTU must be greater than 576 bytes and less than 9214.");
```
Comment (fichtner) [inline src/www/interfaces_lagg_edit.php]: same here

**Comment by darkfires** [comment]
> Please no shell stuff in this PR
I actually didn't mean to do that at all.. don't even understand how that happened

**Comment by fabianfrz** [comment]
Just do a git revert commit I'd and it will be gone again after a push.

However I like the idea of having vtysh in the menu.

**Comment by AdSchellevis** [comment]
cherry-picked in https://github.com/opnsense/core/commit/13f8ba23478e6b3f7cbfc17f594284f2bd3f1b81

**Pull request closed without merging**